heatpump.yaml

```diff
@@ -3257,7 +3202,7 @@ number:
     unit_of_measurement: "°C"
     entity_category: config
     min_value: 20
-    max_value: 60
+    max_value: 65
```

Comment:
The Airwell manual states this as 60, Midea as 75, but 75 is only possible with a external (tank) heater, since R32 is not able to heat until 75.

I am okay with 65